TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteFragment.kt

```diff
@@ -116,4 +134,44 @@ class NoteFragment : Fragment() {
             copyDrawBitmap()
         }
     }
+
+    private fun save() {
+        arguments?.let {
+            val path: String? = requireContext().getExternalFilesDir(null)?.absolutePath
+            val file = File(path + "/" + it.getString("note"))
```

Comment:
if I'm interpreting this correctly, the drawing is saved based on the note title, right? that's definitely fine for now, but we may want to talk about how else we can distinguish between them (because right now in the model, the default title is "untitled" and it would be possible to have multiple notes with the same title

Reply:
same as the first comment, this is just a placeholder for now